rushstack-internal-rigs/heft-rushstack-internal-node-rig/profiles/default/config/rush-project.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "@rushstack/heft-node-rig/profiles/default/config/rush-project.json",
+
+  "operationSettings": [
+    {
+      "operationName": "_phase:build",
+      "outputFolderNames": [".heft", "release"]
```

Comment:
What is `release`? These projects build into the `dist` folder, right?

And what about `temp` and `lib`?

Reply:
`temp` and `lib` come from the rig that we're extending (`@rushstack/heft-node-rig`). Not sure why I included `release` in there. Removed.